src/driver/src/DefaultDriver.cpp

```diff
@@ -167,27 +173,6 @@ genny::driver::DefaultDriver::OutcomeCode genny::driver::DefaultDriver::run(
 }
 
 
-namespace {
-
-/**
- * Normalize the metrics output file command-line option.
- *
- * @param str the input option value from the command-lien
- * @return the file-path that should be used to open the output stream.
- */
-// There may be a more conventional way to define conversion/normalization
-// functions for use with boost::program_options. The tutorial isn't the
-// clearest thing. If we need to do more than 1-2, look into that further.
-std::string normalizeOutputFile(const std::string& str) {
-    if (str == "-") {
-        return std::string("/dev/stdout");
-    }
-    return str;
-}
-
-}  // namespace
```

Comment:
this still has value at least for me when running locally (specifying `-` as an output file name)